mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -456,10 +460,18 @@ ConvolutionDims ConvolutionDims::fromOp(Operation *op) {
       ho = outputShape[i];
     } else if (outputAttr.getValue() == "1o") {
       wo = outputShape[i];
+    } else if (outputAttr.getValue() == "2o") {
+      dout = outputShape[i];
     }
   }
 
-  return ConvolutionDims({y, x}, {ho, wo}, {hi, wi}, k, c, n, g);
+  SmallVector<int64_t> fil({y, x});
```

Comment:
Can't create simple initialiser lists because of the variable size.  Thus temp vectors created and augmented if necessary.